src/modules/common/renderHtml.js

```diff
@@ -32,18 +33,91 @@ const renderJS = (files: FilesStateType) => `
       item.src = '${URL_PREFIX}' + newSrc
     }
   }
+})();
+
+(function() {
+  var container = document.getElementById('measure-container')
```

Comment:
Can we use the body here?

Reply:
no, for some reason, body seems to behave indeterminately (at least for me)